ts/editor/PlainTextBadge.svelte

```diff
@@ -40,6 +41,9 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     class:visible={show || !animated}
     class:highlighted={!off}
     on:click|stopPropagation={toggle}
+    on:keydown={onEnterOrSpace(() => toggle())}
```

Comment:
`on:keydown={undefined}` will work as well for the purpose of fixing the warning.